File: Programmers/두 개 뽑아서 더하기.kt
class Solution {
fun solution(numbers: IntArray): IntArray {
    var data = ArrayDeque<Int>()
    numbers.sort()
    for (i in 0 until numbers.size-1) {
        for (j in i+1 until numbers.size) {
            var save = numbers[i]+numbers[j]
            if (!data.contains(save)) {
                data.addLast(save)
            }
        }
    }
    var resultArr = IntArray(data.size)
    for (i in 0 until data.size) {
        resultArr[i] = data.elementAt(i)
    }
    return resultArr
}
}

File: Programmers/두 개 뽑아서 더하기.py
def solution(numbers):
    basket = set()
    n_sum = 0
    for i in range(len(numbers)-1):
        for j in range(i+1, len(numbers)):
            n_sum = numbers[i] + numbers[j]
            if n_sum not in basket:
                basket.add(n_sum)
    result_arr = list(basket)
    return sorted(result_arr)
